snaps/tutorials/transaction-insights.md

```diff
@@ -12,22 +12,20 @@ The Snap provides transaction insights in the MetaMask transaction window.
 ## Prerequisites
 
 - [MetaMask Flask installed](../get-started/install-flask.md)
-
 - An account on your MetaMask Flask instance with testnet ETH
-
   :::tip
   You can use [Infura's Sepolia faucet](https://www.infura.io/faucet) to get Sepolia ETH.
   :::
-
 - A text editor (for example, [VS Code](https://code.visualstudio.com/))
-
+- [Node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 18.16 or later
 - [Yarn](https://yarnpkg.com/) version 3
```

Comment:
I actually don't know why we say version 3 here, the create command only works in version 1....

Reply:
It works with my Yarn version 3, let's just say "Yarn" as the requirement.